commercial/app/views/travel/travelProminent.scala.html

```diff
@@ -1,20 +1,18 @@
 @(offers: Seq[model.commercial.travel.TravelOffer], omnitureId: Option[String], clickMacro: Option[String])(implicit request: RequestHeader)
 
-@import conf.Configuration
-
 @defining(("1_1", "2014-07-04")) { case (version, date) =>
     <div class="commercial commercial--travel commercial--tone-travel commercial--high commercial--prominent" data-link-name="commercial-high | travel | @omnitureId">
         <div class="commercial__inner">
             <div class="commercial__header">
                 <h3 class="commercial__title">
-                    <a href="@clickMacro@Configuration.commercial.travel_url/" data-link-name="merchandising-travel-v@{version}_@{date}-high-title">
+                    <a href="@{clickMacro}http://www.guardianholidayoffers.co.uk/" data-link-name="merchandising-travel-v@{version}_@{date}-high-title">
```

Comment:
This URL doesn't actually work.  I just stuck it there to get rid of the property.  The new feed has complete target URLs so this will go altogether in the next step.